modules/imaging_browser/README.md

```diff
@@ -92,3 +92,5 @@ For downloading large DICOM files, it may be necessary to increase the
   as configured by the study.
 - The control panel on the "View Session" page includes links to the DICOM Archive 
   and download capabilities for any DICOM tars associated with the given session.
+  Note that DICOM downloads will be prepended with the Patient Name field though
+  are not actually stored this way on the filesystem.
```

Comment:
super minor: missing they in `though "they" are`

Reply:
@christinerogers Thanks! All done. Can you update your review?

Reply:
> There is one instance in each file.